test/api_bridge_test.rb

```diff
@@ -140,25 +152,36 @@ def before_setup
     assert_equal @f_app, @api.prepare_options({})["f"]
 
     # set a new filter which is not the same as the app wide filter
-    assert_equal ["collection|letters", "keyword|production", "subcategory|memos"], @api.prepare_options({ "f" => ["subcategory|memos"] })["f"]
-
-    # if a filter collides (collection in this case), make sure the newly requested one wins out!
     assert_equal(
-      ["keyword|production", "collection|whitman", "subcategory|marginalia"],
-      @api.prepare_options({ "f" => ["collection|whitman", "subcategory|marginalia"] })["f"]
+      ["collection|letters", "keyword|production", "subcategory|memos"],
```

Comment:
I suppose for even more de-duplicating, we could write this as `@f_app + ["subcategory|memos"]`. That mimics the code being tested that concatenates the array contents too. What do you think? :thinking:

Reply:
That's true. I can do either, I don't have a particular preference!

Reply:
It'll also simplify rewriting test values if we need to change `@f_app` for future tests. Give it a try?